.travis.yml

```diff
@@ -1,18 +1,19 @@
+bundler_args: --without development --retry=3 --jobs=3
+cache: bundler
 language: ruby
 rvm:
+  - rbx-2
+  - jruby
+  - jruby-head
+  - ruby-head
   - 1.9.2
   - 1.9.3
   - 2.0.0
   - 2.1
   - 2.2
-  - jruby
-  - jruby-head
-  - rbx-2
-  - ruby-head
 matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: jruby-head
-before_install:
- - sudo apt-get update
- - sudo apt-get install idn
```

Comment:
This `idn` package appears to be part of the default Travis Docker image, since the build is passing without it. If that ever changes and it needs to be added back, it can be done without `sudo` suing the [APT add-on](http://docs.travis-ci.com/user/apt/).